grobid-core/src/main/java/org/grobid/core/engines/label/TaggingLabels.java

```diff
@@ -15,14 +15,28 @@
 
     protected static final ConcurrentMap<Pair<GrobidModel, String>, TaggingLabel> cache = new ConcurrentHashMap<>();
 
+    //IOB labels and prefixes
+    public static final String START_ENTITY_LABEL_PREFIX_IOB_BEGINNING = "B-";
+    public static final String START_ENTITY_LABEL_PREFIX_IOB_INSIDE = "I-";
```

Comment:
Naming confusing because of `START_ENTITY` and `INSIDE` or`START_ENTITY` plus redundant `BEGINNING` . We could have:

```
public static final String IOB_START_ENTITY_LABEL_PREFIX = "B-";
public static final String IOB_INSIDE_ENTITY_LABEL_PREFIX = "I-";
public static final String IOB_OTHER_LABEL = "O";

public static final String GROBID_START_ENTITY_LABEL_PREFIX = "I-";
public static final String GROBID_INSIDE_ENTITY_LABEL_PREFIX = "";
public final static String GROBID_OTHER_LABEL = "<other>";

public static final String ENAMEX_START_ENTITY_LABEL_PREFIX = "E-";
```

Reply:
Thanks! I pushed the change as requested 😉